toonz/sources/CMakeLists.txt

```diff
@@ -48,6 +49,26 @@ elseif (APPLE)
     else()
         message(FATAL_ERROR "Invalid PLATFORM:" ${PLATFORM} ". 'PLATFORM' must be 32 or 64.")
     endif()
+elseif(UNIX)
+    message("Unix System")
+    set(PLATFORM 64)  # XXX
+    set(LIB_SUFFIX "so")
+    add_definitions(-DLINUX)
+    set(CMAKE_CXX_STANDARD 11)
+    include_directories(
+        SYSTEM
+        /usr/include/qt
+        /usr/include/qt/QtCore
+        /usr/include/qt/QtWidgets
```

Comment:
Right, I should have added `FIXME` comment there. It should of course use qt's finder.